tools/who_what_benchmark/tests/test_cli_text.py

```diff
@@ -239,3 +239,29 @@ def test_text_genai_cb_model(tmp_path):
     ])
     assert "Metrics for model" in output
     assert "## Reference text" not in output
+
+
+def test_text_genai_json_string_config():
+    if sys.platform == 'darwin':
+        pytest.xfail("Ticket 173169")
+
+    cb_json_string = "{\"max_num_batched_tokens\": 4096}"
+    ov_json_string = "{\"KV_CACHE_PRECISION\":\"f16\"}"
+
+    output = run_wwb([
+        "--base-model",
+        base_model_path,
+        "--target-model",
+        target_model_path,
+        "--num-samples",
+        "2",
+        "--device",
+        "CPU",
+        "--genai",
+        "--cb-config",
+        cb_json_string,
+        "--ov-config",
+        ov_json_string
```

Comment:
Is there a test covering the case when --ov-config points to a file?
We tried passing a JSON file with {"ATTENTION_BACKEND": "SDPA"} inside, but it doesn't seem to work as expected.

Reply:
> It is not covered by test case, what is the error?

The issue appeared to be related not to loading parameters from the ov_config file but to exporting using `--task text-generation`, instead of text-generation-with-past. More details are available in ticket 175999

But still, it would be great to have a test of old behavior using a JSON file, as @sbalandi suggested.